sample_apps/simple_app_under_oe/instructions.md

```diff
@@ -157,15 +157,22 @@ $CERTIFIER_PROTOTYPE/utilities/package_claims.exe     \
 $CERTIFIER_PROTOTYPE/utilities/print_packaged_claims.exe --input=policy.bin
 ```
 
-## Step 8: Use Policy Generator
+## Step 8: Use Automated Policy Generator
 
 ### a. Edit policy file
 
 Open `simple_app_under_oe/oe_policy.json` and replace trusted measurements
 in the "measurements" property with the expected measurement from
 `make dump_mrenclave`.
 
-### b. Run Policy Generator
+### b. Build the Policy Generator
```

Comment:
You can see the updated [instructions.md file here](https://github.com/vmware-research/certifier-framework-for-confidential-computing/blob/agurajada/82-oe-regression/sample_apps/simple_app_under_oe/instructions.md).